File: c/04-vetores/main076.c
/*Crie um programa que preencha automaticamente um vetor numérico com 7 
números gerados aleatoriamente pelo computador e depois mostre os valores 
gerados na tela.

Data 31/10/2025 -> 38ª Dia
*/

#include <stdio.h>
#include <stdlib.h>
#include <time.h>

int main(){

    int vetor[7];

    srand(time(NULL));

    for(int i = 0; i < 7; i++){

        vetor[i] = rand() % 7;
    }
    for(int i = 0; i < 7; i++){

        printf("%d ",vetor[i]);
    }
    return 0;

}

File: python/04-lista/main076.py
'''Crie um programa que preencha automaticamente um vetor numérico com 7 
números gerados aleatoriamente pelo computador e depois mostre os valores 
gerados na tela.

Data 31/10/2025 -> 38ª Dia
'''

import random

lista = []

for i in range(7):

    lista.append(random.randint(1,7))

print(lista)